WeatherMax.py

```diff
@@ -0,0 +1,148 @@
+def tempLists(filName):
```

Comment:
What does `filName`mean? It could have been `fileName` but that too is a poor name.
The naming of variables is very important. Names should be descriptive and self-explanatory.

Reply:
The same goes for your filename which is `WeatherMax.py` task is `weatherman`.

Reply:
Poor function name, what does it describe? Avoid using words like `lists`, `data`, or `temp` in naming variables or functions.

Reply:
Python code should be `PEP8` compliant. Read carefully what `PEP8` asks you to follow.
As per `PEP8` variable or function naming should be like [this](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).